docs/sources/panels/inspect-panel.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Inspect a panel"
+type = "docs"
+[menu.docs]
+identifier = "inspect-a-panel"
+parent = "panels"
+weight = 400
++++
+
+# Inspect a panel
+
+> **Note:** This documentation refers to a feature only available in Grafana 7.0 beta.
+
+The panel inspector helps you understand and troubleshoot your panels. You can inspect the raw data for any Grafana panel, export that data to a comma-separated variable (CSV) file, view query requests, and export panel and data JSON.
+
+## Panel inspector UI
+
+The panel inspector displays Inspect: <NameOfPanelBeingInspected> at the top of the pane. Click the arrow in the upper right corner to expand or reduce the pane.
```

Comment:
Reduce? I'm not native and this feels weird to me. I would expect expand/collapse. On the other hand it's not actually collapsing. I trust your mother tongue @oddlittlebird :)